docs/tutorials/_tabs/setting-up-ethernet-over-bluetooth-on-ubuntu-14.04.md

```diff
@@ -30,3 +29,16 @@ If you are using an older version, please upgrade.
     depend on the Bluetooth adapter on your host computer.
 
     {% include screenshot.html source="/images/brickman/bluetooth-confirm-passkey.png" %}
+
+---
+
+These instructions use blueman. Install it with sudo apt-get install blueman. Blueman should work parallel to Network Manager, too. I assume you have already paired your PC with your EV3 brick using the discreption above.
+
+Using blueman it is quite easy to establish an ethernet connection with internet.
+
+1.  Start blueman (if not already started)
+2.  You should see a bluetooth icon in the task bar. Left click on it and choose "local services"
+3.  Go to network. Ensure you activated "Network Access Point (NAP)" and "dnsmasq". For "PAN support" choose "NetworkManager" but for DUN-support "Blueman".
+4.  Close this window, and grab you EV3 brick.
+5.  In the Bluetooth menu entry choose your PC under "Devices" and click "Connect".
```

Comment:
I did not see this on the PC, but I was able to successfully connect by opening the PC bluetooth device on the EV3 and selecting "Network Connection" to take me to the networking screen, then I selected "Connect" on the "Conn." tab.